config/lpdc-management/characteristics/form.ttl

```diff
@@ -191,8 +193,52 @@ ext:authorityPg a form:PropertyGroup;
       form:displayType displayTypes:conceptSelector ;
       sh:group ext:authorityPg .
 
+
+  ##########################################################
+  # Conditionally show the required or not-required variant of the uitvoerende overheid field 
+  ##########################################################
+  ext:temp-fg-a a form:FieldGroup .
+  ext:temp-fg-a form:hasField ext:executingAuthorityLevelF .
+
+  ext:temp-fg-b a form:FieldGroup .
+  ext:temp-fg-b form:hasField ext:executingAuthorityLevelF .
+
+  ext:executingAuthorityLevelF form:hasConditionalFieldGroup ext:temp-a.
+  ext:executingAuthorityLevelF form:hasConditionalFieldGroup ext:temp-b.
+
+
+  ext:temp-a a form:ConditionalFieldGroup ;
+      form:conditions
+        [ a form:MatchValues ;
+          form:grouping form:Bag ;
+          sh:path lpdcExt:executingAuthorityLevel ;
+          # Tip: you can use form:valuesNotIn to exclude from a list (if triples are found matching the path)
+          form:valuesNotIn (
+            <https://productencatalogus.data.vlaanderen.be/id/concept/UitvoerendBestuursniveau/Derden>
+          );
+        ] ;
+      form:hasFieldGroup ext:temp-fg-a-result .
+  
+  ext:temp-b a form:ConditionalFieldGroup ;
+      form:conditions
+      [ a form:SingleCodelistValue ;
```

Comment:
SingleCodeListValue doesn't seem to work because we use the "conceptSelector" field so the code list values aren't stored in the meta graph.

Reply:
Maybe this one works? https://github.com/lblod/submission-form-helpers/blob/master/src/constraints/exact-value.js